migration_toolkit_for_containers/mtc-migrating-vms.adoc

```diff
@@ -0,0 +1,77 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="mtc-migrating-vms"]
+= Migrating Virtual machine storage
+include::_attributes/common-attributes.adoc[]
+:context: mtc-migrating-vms
+
+toc::[]
+
+{mtc-full} (mtc-short) 1.8.4 supports storage migration for KubeVirt virtual machines (VMs).
+
+The migration can occur offline, when the VM is turned off, or online, when the VM is running.
+
+The main use case for this storage migration for KubeVirt VMs is migration from one storage class to another. For example, if you are rebalancing between different storage providers or if there is new storage available that is better suited to the workload running inside the VM.
+
+== Supported persistent volume Actions
+
+[width="100%",cols="31%,33%,36%",options="header",]
+|===
+|Action
+|Supported
+|Description
+
+|Copy
+|Yes
+a|
+. Create a new persistent volume (PV) in the same namespace.
+. Copy data from source PV to target PV, and modify the VM definition to point to the new PV.
+** If the `liveMigrate` flag is set, the VM live migrate.
+** If the `liveMigrate` flag is not set, the VM shuts down, and the source PV contents are copied to the target PV and the VM is restarted.
+
+|Move
+|No
+|This action is not supported.
+|===
+
+== Preparing for migration
+
+To prepare for migration, deploy mig-controller and Velero, and then create the following `Mig`' resources on the cluster where the migration controller is running to prepare for migration:
+
```

Comment:
suggestions: `mig-controller`  and "the following `mig` resources.." there's an extra character after `Mig` .